app/src/main/java/com/google/droidjump/DroidStartView.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.droidjump;
+
+import static android.graphics.Bitmap.createBitmap;
+import static android.graphics.BitmapFactory.decodeResource;
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.view.View;
+
+public class DroidStartView extends View {
+
+    public DroidStartView(Context context) {
+        super(context);
+    }
+
+    @Override
+    protected void onDraw(Canvas canvas) {
+        super.onDraw(canvas);
+        Bitmap fullDroidPic = decodeResource(getResources(), R.mipmap.droid);
+        int droidCountOnFullDroidPic = 9;
+        int droidStep = (int) fullDroidPic.getWidth() / droidCountOnFullDroidPic;
+        int droidCharacterStandingNumber = 4;
```

Comment:
Could we create an enum/class with constants with these numbers like "droidCountOnFullDroidPic" and "droidCharacterStandingNumber"? I see that these values and other values are used in several of your pull requests.

Reply:
I suggest using an interface that will be great here. What I found about constants in Java
https://www.informit.com/articles/article.aspx?p=331398&seqNum=5#:~:text=Java%20programmers%20commonly%20define%20constants,they%20are%20static%20and%20final.